conformance/06_tags_list_test.go

```diff
@@ -51,14 +51,19 @@ var test06TagsList = func() {
 			err = json.Unmarshal(jsonData, tagList)
 			Expect(err).To(BeNil())
 			Expect(len(tagList.Tags)).To(Equal(numResults))
-			lastTagList = *tagList
+			lastTagList = &TagList{
+				Name: tagList.Name,
+				Tags: tagList.Tags,
+			}
 		})
 
 		g.Specify("GET start of tag is set by `last` query parameter", func() {
 			numResults := numTags / 2
+			Expect(len(lastTagList.Tags)).To(BeNumerically(">=", 0))
```

Comment:
did you want this to be ok with 0?

Reply:
This got folded into our other PR and will be addressed there. In the meantime, I'm going to close this PR